Projects/ChangeTier/Sources/Service/Protocols/ChangeTierClient.swift

```diff
@@ -0,0 +1,3 @@
+public protocol ChangeTierClient {
+    func getTier(contractId: String) async throws -> ChangeTierIntentModel
```

Comment:
Also, we need to pass ChangeTierSource also here since it would affect response that we will get for the  ChangeTierIntentModel?